django/contrib/admin/views/autocomplete.py

```diff
@@ -26,12 +26,21 @@ def get(self, request, *args, **kwargs):
         context = self.get_context_data()
         return JsonResponse({
```

Comment:
The docstring on lines 14-18 is no longer, strictly speaking, always true. Maybe it could be updated to say something like:
"Return (by default) a JsonResponse with search results of the form:".

I feel like my suggestion here could be better phrased, but I'm not sure how at this moment.